CLI_QUICKSTART.markdown

```diff
@@ -1,29 +1,28 @@
-# A quick start using the Veracruz command-line interface
+# Getting started with the Veracruz command-line interface
 
-This document is a quick overview on how to use the Veracruz command-line
-interface using artifacts built during testing as a part of the
-test-collateral.
+This document walks through running an example Veracruz program using
+the Veracruz command-line interface.
 
-For a more in-depth guide on how to use Veracruz and how to manually generate
-the necessary artifacts, see [CLI_INSTRUCTIONS.markdown](CLI_INSTRUCTIONS.markdown).
+It assumes you have walked through the steps in [BUILD_INSTRUCTIONS.markdown](BUILD_INSTRUCTIONS.markdown).
 
-This document assumes you have walked through the steps in
-[BUILD_INSTRUCTIONS.markdown](BUILD_INSTRUCTIONS.markdown), and have a working
-build environment.
+So do that first and make sure the tests are passing! The rest of this
+walkthrough will assume you're in a docker container capable of building
+Veracruz.
 
 ## Building the Veracruz binaries
 
-First thing is we need to build the actual Veracruz command-line interfaces,
-a WebAssembly toolchain, and the test-collateral artifacts. These can be
-compiled using make, however we need to specify which Trusted Execution
-Environment (TEE) we are building for. So for example, if you are running
-Veracruz on Nitro, you would run `make nitro-cli-install`.
+First lets make sure we've built and installed the Veracruz binaries. For
```

Comment:
let's

Reply:
I've went ahead and did a `%s/lets/let's/g` on this file :)